dataflow/src/main/java/org/checkerframework/dataflow/analysis/BackwardTransferFunction.java

```diff
@@ -0,0 +1,53 @@
+package org.checkerframework.dataflow.analysis;
+
+import java.util.List;
+import org.checkerframework.checker.nullness.qual.Nullable;
+import org.checkerframework.dataflow.cfg.UnderlyingAST;
+import org.checkerframework.dataflow.cfg.node.Node;
+import org.checkerframework.dataflow.cfg.node.ReturnNode;
+
+/**
+ * Interface of a backward transfer function for the abstract interpretation used for the backward
+ * flow analysis.
+ *
+ * <p>A backward transfer function consists of the following components:
+ *
+ * <ul>
+ *   <li>A method {@code initialNormalExitStore} that determines which initial store should be used
+ *       at the normal exit block in the org.checkerframework.dataflow backward analysis.
+ *   <li>A method {@code initialExceptionalExitStore} that determines which initial store should be
+ *       used at the exceptional exit block in the org.checkerframework.dataflow backward analysis.
+ *   <li>A function for every {@link Node} type that determines the behavior of the
```

Comment:
The first two bullet points are methods in this interface. This bullet point is suddenly a `function` and no such methods are defined in the interface. Can you clarify this?